services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -877,7 +903,10 @@ protected void initBlockBlobSidecarsTrackersPool() {
               recentChainData,
               executionLayer,
               () -> blobSidecarValidator,
-              blobSidecarGossipChannel::publishBlobSidecar);
+              blobSidecarGossipChannel::publishBlobSidecar,
+              isSuperNode,
+              () -> kzg,
```

Comment:
nit: Not sure why we need a supplier here? Why we couldn't just pass a `KZG` instance?